docs/sdk-and-tools/indices/accounts.md

```diff
@@ -0,0 +1,58 @@
+---
+id: es-index-accounts
+title: Elasticsearch index: accounts
+---
+
+The `_id` field for this index is composed of bech32 encoded address.
+
+| Field      | Description                                                                                                |
+|------------|------------------------------------------------------------------------------------------------------------|
+| address    | The address field holds the address bytes in a bech32 encoding.                                            |
+| balance    | The balance field holds the amount of EGLD the address has.                                                |
```

Comment:
The balance field holds the amount of EGLD the address possesses. It is a string that also includes the number of decimals. Example: `"1500000000000000000"` (equivalent to 1.5 EGLD).

Reply:
done